src/trilateration/trilateration.cpp

```diff
@@ -0,0 +1,186 @@
+/**
+ * Hydrophone Trilateration (Position Derivation) Node
+ *
+ * @author Ryan Summers
+ * @date 12-12-2016
+ *
+ * @brief This file contains the calculations of the submarine's position
+ *        relative to the pinger from a number of hydrophone time stamps.
+ *
+ * @note The mathematic equations presented in this file are derived at the
+ *       following two links. Please refer to them for all questions relating
+ *       to mathematic operations. All variables have been named after the
+ *       variables defined in the derivation for clarity.
+ *          http://robosub.eecs.wsu.edu/wiki/ee/hydrophones/trilateration/start
+ *          http://robosub.eecs.wsu.edu/wiki/cs/hydrophones/start
+ */
+
+#include "robosub/HydrophoneDeltas.h"
+#include "robosub/Position.h"
+#include "ros/ros.h"
+
+ros::Publisher position_pub;
+
+/*
+ * The speed of sound in water is 1484 m/s.
+ */
+static constexpr double speed_sound_in_water = 1484;
```

Comment:
I would agree if it were a #define, however this is a double variable type, therefore I would not classify it as all caps personally. I don't like the all caps syntax.